drivers/input/touchscreen/rpi-ft5406.c

```diff
@@ -61,13 +60,14 @@ static int ft5406_thread(void *arg)
 	struct ft5406 *ts = (struct ft5406 *) arg;
 	struct ft5406_regs regs;
 	int known_ids = 0;
-	
+        uint8_t temp = 99;	
+
 	while(!kthread_should_stop())
 	{
 		// 60fps polling
 		msleep_interruptible(17);
-		memcpy_fromio(&regs, ts->regs, sizeof(*ts->regs));
-		writel(99, &ts->regs->num_points);
+		memcpy_fromio(&regs, ts->ts_base, sizeof(regs));
```

Comment:
I think for now minimising the lines of code changed makes it easier to verify things won't break.
I'd suggest leaving "cosmetic" commits out for now.
They can always be submitted in a subsequent PR if you think it improves readability/consistency.